avaframe/runAna5DFAPathGeneration.py

```diff
@@ -0,0 +1,150 @@
+        outDir = pathlib.Path(avalancheDir, 'Outputs', 'ana5Utils', 'DFAPath')
+        fU.makeADir(outDir)
+        # write configuration to file
+        com1DFACfgFile = outDir / 'com1DFAPathGenerationCfg.ini'
+        with open(com1DFACfgFile, 'w') as configfile:
+            com1DFACfg.write(configfile)
+        # call com1DFA and perform simulations
+        dem, plotDict, reportDictList, simDF = com1DFA.com1DFAMain(cfgMain, cfgInfo=com1DFACfgFile)
+    else:
+        # read simulation dem
+        demOri = gI.readDEM(avalancheDir)
+        dem = com1DFA.setDEMoriginToZero(demOri)
+        dem['originalHeader'] = demOri['header'].copy()
+        # load DFA results (use runCom1DFA to generate these results for example)
+        # here is an example with com1DFA but another DFA computational module can be used
+        # as long as it produces some particles or FV, FT and FM results
+        # create dataFrame of results (read DFA data)
+        simDF, _ = cfgUtils.readAllConfigurationInfo(avalancheDir)
+
+    #Clean DFAPath output in avalanche directory
+    initProj.cleanModuleFiles(avalancheDir, DFAPath, deleteOutput=True)
+
+    for simName, simDFrow in simDF.iterrows():
+        log.info('Computing avalanche path from simulation: %s', simName)
+        pathFromPart = cfgDFAPath['PATH'].getboolean('pathFromPart')
+        resampleDistance = cfgDFAPath['PATH'].getfloat('nCellsResample') * dem['header']['cellsize']
+        # get the mass average path
+        avaProfileMass, particlesIni = DFAPath.generateAveragePath(avalancheDir, pathFromPart, simName, dem,
+                                                                   addVelocityInfo=cfgDFAPath['PATH'].getboolean('addVelocityInfo'))
+        avaProfileMass, _ = gT.prepareLine(dem, avaProfileMass, distance=resampleDistance, Point=None)
+        avaProfileMass['indStartMassAverage'] = 1
+        avaProfileMass['indEndMassAverage'] = np.size(avaProfileMass['x'])
+        # make the parabolic fit
+        parabolicFit = DFAPath.getParabolicFit(cfgDFAPath['PATH'], avaProfileMass, dem)
+        # here the avaProfileMass given in input is overwritten and returns only an x, y, z extended profile
+        avaProfileMass = DFAPath.extendDFAPath(cfgDFAPath['PATH'], avaProfileMass, dem, particlesIni)
+        # resample path and keep track of start and end of mass averaged part
+        avaProfileMass = DFAPath.resamplePath(cfgDFAPath['PATH'], dem, avaProfileMass)
+        # get split point
+        splitPoint = DFAPath.getSplitPoint(cfgDFAPath['PATH'], avaProfileMass, parabolicFit)
+        # make analysis and generate plots
+        _ = outCom3Plots.generateCom1DFAPathPlot(avalancheDir, cfgDFAPath['PATH'], avaProfileMass, dem,
+                                                 parabolicFit, splitPoint, simName)
+        # now save the path and split point as shapefiles
+        massAvgPath,splitPoint = DFAPath.saveSplitAndPath(avalancheDir, simDFrow, splitPoint, avaProfileMass, dem)
```

Comment:
Now we are moving this file away from an example runscript, I think we should move this part as a function to the ana5 DFAPathGeneration.py file (similar to com1DFAMain). 
-> keep the runscript smaller
-> have a function to be called directly